.gitignore

```diff
@@ -8,3 +8,7 @@ resources/
 # Link checker artifacts
 bin/
 tmp/
+
+# Local Netlify folder
+.netlify
```

Comment:
Do you actually get a `.netlify` folder created locally?

Reply:
Yes, when you use the `netlify-cli`, and link it to a site, it generates this folder.

Reply:
Ah, I've been using netlify-cli but it hasn't created such a folder. As you say, probably because you're linking to a site.
(Topic for another day: why would you need to link to a site?)

Reply:
it gives you the ability to do manual uploads, as well as get some additional debugging info